### PR TITLE
Update README Based on Roadmap. Remove Stale Information

### PR DESCRIPTION
**Pull request by allella**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `master` branch of Chapter.

Closes: Nothing

- Removed Elasticsearch, since Quincy agreed with #32 that no advanced search tech is needed for the early going.
- Removed the stale user stories and referenced the live Github MVP project and #84 for post MVP.
- Remove stale roadmap bullets and linked to the live #47, which has an up-to-date list at the top.
- Added suggested steps before creating a new user story issue.
- Tweaked Docker link to be less verbose.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,76 +109,35 @@ npm run test:watch
...
+Quincy Larson is the project lead. [FreeCodeCamp](https://www.freecodecamp.org) will start "dogfooding" the MVP with several of its local study groups.
 
 Here's an out-dated example of an app with similar functionality: [The freeCodeCamp Study Group Directory](https://study-group-directory.freecodecamp.org).
 
-You should [join our Discord server](https://discord.gg/vbRUYWS) to get connected with people interested in this project and to be aware of our future announcements.
 
 ## Contributing
 
-[**Take part in discussions or contribute code to this opensource codebase.**](CONTRIBUTING.md)
+* You should [join our Discord server](https://discord.gg/vbRUYWS) to get connected with people interested in this project and to be aware of our future announcements.
```
Comment (vkWeb) [inline README.md]: Not required.

File: README.md [hunk 2]
```diff
@@ -110,76 +109,35 @@ npm run test:watch
...
+* Please read the [**suggested steps to contribute code to the Chapter project**](CONTRIBUTING.md) below forking or submitting any pull requests.
 
 ## License
 
 Copyright © 2019 freeCodeCamp.org
 
-The content of this repository is bound by the following license(s):
-
-- The computer software is licensed under the [BSD-3-Clause](LICENSE) license.
+The computer software is licensed under the [BSD-3-Clause](LICENSE) license.
```
Comment (vkWeb) [inline README.md]: Revert to old one.

File: README.md [hunk 3]
```diff
@@ -110,76 +109,35 @@ npm run test:watch
...
-- I can check-in attendees on the event registration desk with their `email_id` or `chapter_id`.
+1. Please check your idea does not already exist as an [MVP issue](https://github.com/freeCodeCamp/chapter/labels/MVP), or in the [post-MVP list](https://github.com/freeCodeCamp/chapter/issues/84).
+1. Create a new "Issue" and post details about the suggested user story.
 
 ## Roadmap
 
-1. Design the schema.
-2. Set up the API endpoints.
-3. Build the web client and let other developers use the API to build mobile clients and voice interface clients.
+The on-going [project Roadmap conversation](https://github.com/freeCodeCamp/chapter/issues/47) is regularly updated to reflect the overall progress and for higher-level discussions.
```
Comment (vkWeb) [inline README.md]: Add this below the old roadmap. Something like:

```suggestion
1. Design the schema - Done.
2. Set up the API endpoints - Done.
3. Built the web client and let other developers use the API to build mobile clients and voice interface clients.

We have a more detailed information on roadmap on the [Roadmap issue]https://github.com/freeCodeCamp/chapter/issues/47). It is regularly updated to reflect the overall progress and for higher-level discussions.
```

File: README.md [hunk 4]
```diff
@@ -110,76 +109,35 @@ npm run test:watch
...
-## User stories so far
+## User Stories
 
-Our goal is to keep things simple and not reinvent wheels. So far we have only two user roles: participants and chapter organizers, both of which are users.
+### MVP
+The [MVP user stories are shown in the MVP Project](https://github.com/freeCodeCamp/chapter/projects/1) kanban / cards and as [issues marked with "MVP"](https://github.com/freeCodeCamp/chapter/labels/MVP).
 
-### As a user
+### Post-MVP
+We are maintaining a list of [post-MVP user stories](https://github.com/freeCodeCamp/chapter/issues/84).
```
[the remark this answers is not in the document]
Comment (allella) [reply README.md]: @vkWeb this is such a small concern, but from a technical writing standpoint we should never have a single bullet point. If/when a second bullet is added then we can certainly add bullets.

File: README.md [hunk 5]
```diff
@@ -110,76 +109,35 @@ npm run test:watch
...
-
-- I can add a venue sponsor to the event with a link to their website as a way of thanking them for hosting.
-
-- I can add a food sponsor to the event with a link to their website as a way of thanking them for food.
-
-- I can see how many times a participant has come to the event as well as their attendance rate.
-
-- I can check-in attendees on the event registration desk with their `email_id` or `chapter_id`.
+1. Please check your idea does not already exist as an [MVP issue](https://github.com/freeCodeCamp/chapter/labels/MVP), or in the [post-MVP list](https://github.com/freeCodeCamp/chapter/issues/84).
+1. Create a new "Issue" and post details about the suggested user story.
```
[the remark this answers is not in the document]
Comment (allella) [reply README.md]: I'd be glad to move this to CONTRIBUTING.md if other find it helpful, but I'm not going to maintain that in needs to be in the README.

File: README.md (hunk 1 above)
Comment (allella) [reply README.md]: This was previously in the README and I moved it. If we want to move it to CONTRIBUTING.md then that's fine with me.

File: README.md (hunk 2 above)
Comment (allella) [reply README.md]: We should avoid single bullet points, so I'm going to suggest we remove the bullet.

File: README.md (hunk 3 above)
Comment (allella) [reply README.md]: If someone else is going to maintain the bullets here then I'm fine with it. As it was, this list was out of sync with #47, which has the established roadmap and is much easier / faster to edit.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (allella) [reply README.md]: Sorry, I meant to say that the user stories were well out of sync with the other two MVP documents.  I don't see why we'd want to maintain the same information in 2 or 3 places unless someone is going to actively maintain all of the locations.

It's already enough work just to keep things up to date so my vote is for more links and less of a huge README that gets out of date every day.

File: README.md [hunk 6]
```diff
@@ -31,7 +31,6 @@ We are planning to use the following tools:
 
 * [Node.js](https://nodejs.org) / [Express](https://expressjs.com) for our backend using JavaScript/TypeScript
 * [Postgres](https://www.postgresql.org) with [Sequelize ORM](https://sequelize.org)
-* [Elasticsearch](https://www.elastic.co/what-is/elasticsearch)
 * [Next.js](https://nextjs.org/) for both client and server-side rendering of the frontend (NextJS is based on [React](https://reactjs.org))
   * [JavaScript/TypeScript](https://www.typescriptlang.org/index.html#download-links)
   * [Styled Components](https://www.styled-components.com) for styling.
```
[the remark this answers is not in the document]
Comment (allella) [reply README.md]: @vkWeb what is true, but in the last meeting we also decided on Javscript inside of Typescript files to not alienate Javascript contributors.  I'm happy to have this reworded, but it seemed pretty clear we were not 100% Typescript and someone else previously added this bullet point, so it's not really part of this PR.

File: README.md [hunk 7]
```diff
@@ -54,7 +53,7 @@ Ensure you are installing Node 10 or greater and npm 6 or greater.
 
 ### Installing Docker
 
-Click [here](https://docs.docker.com/v17.12/install/) for the Docker installation site.  Scroll down to "Supported Platforms" and follow the instructions to download & install Docker Desktop for your operating system (or Docker CE for linux).
+See the [Docker installation](https://docs.docker.com/v17.12/install/) "Supported Platforms" section and follow the instructions to download & install Docker Desktop for your operating system (or Docker CE for linux).
```
Comment (ScottBrenner) [inline README.md]:
```suggestion
See the [Docker installation](https://docs.docker.com/install/) "Supported platforms" section and follow the instructions to download & install Docker Desktop for your operating system (or Docker CE for linux).
```

Can be fixed in a separate PR, but the current Docker install link here (for v17.12) is out of date - this link ☝️ will always link to the latest release.

File: README.md [hunk 8]
```diff
@@ -110,76 +109,35 @@ npm run test:watch
...
 
 Here's an out-dated example of an app with similar functionality: [The freeCodeCamp Study Group Directory](https://study-group-directory.freecodecamp.org).
 
-You should [join our Discord server](https://discord.gg/vbRUYWS) to get connected with people interested in this project and to be aware of our future announcements.
 
 ## Contributing
 
-[**Take part in discussions or contribute code to this opensource codebase.**](CONTRIBUTING.md)
+* You should [join our Discord server](https://discord.gg/vbRUYWS) to get connected with people interested in this project and to be aware of our future announcements.
+* Please read the [**suggested steps to contribute code to the Chapter project**](CONTRIBUTING.md) below forking or submitting any pull requests.
```
Comment (ScottBrenner) [inline README.md]:
```suggestion
* Please read the [**suggested steps to contribute code to the Chapter project**](CONTRIBUTING.md) before forking or submitting any pull requests.
```

Comment (allella) [reply README.md]: Thanks Scott for the typo fix. @vkWeb had suggested not having this new bullet.  I don't think it's critical and perhaps it's better in the CONTRIBUTING.md.

The purpose was to more to encourage new user story suggestions to create a new issue instead of dropping new ideas in comments, where it's likely to get lose.

The bullet about checking that the idea doesn't already exist was secondary, but still a reminder for anybody who might not know where to look for the full set of existing suggestions.

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (allella) [reply README.md]: We do have a similar reminder in the CONTRIBUTING.md.  So, I'd suggest we update the CONTRIBUTING.md to link to the MVP project and #84 as a quick view of what's already in the works or been discussed.

File: README.md (hunk 1 above)
Comment (allella) [reply README.md]: Discord is similarly mentioned in the CONTRIBUTING.md.

This note already existed in the README, so I have no strong feelings that it needs to be in the README.

However, if we leave it then I'd vote to move it out of the Roadmap section and into the Contributing section.

File: README.md (hunk 7 above)
Comment (allella) [reply README.md]: Good suggestion.

The original change was to reword it to avoid "Click here", so if this is changed in another PR then my preference is to [avoid "Click here" links](https://www.w3.org/QA/Tips/noClickHere).

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (allella) [reply README.md]: Similar to the user stories and roadmap bullets, the Quick Links I added initially in the CONTRIBUTING.md are out of date.

My feeling is many of the bullets we initially added to the README and CONTRIBUTING made sense only temporarily because there was less structure and places to point newcomers in the initial chaos.

If/as other pages in the Github project go into more details it's fair to consider linking out to those sections to avoid huge and/or out of date .md files.

[the remark this answers is not in the document]
Comment (vkWeb) [reply README.md]: @allella Umm.. Yes you are right. 

Let's keep it that way then ;)

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (vkWeb) [reply README.md]: @allella Okay fine, we can keep it in contributing.md.

File: README.md (hunk 1 above)
Comment (vkWeb) [reply README.md]: @allella There's no harm in keeping it in readme's roadmap and contributing. Let it be.

File: README.md (hunk 3 above)
Comment (vkWeb) [reply README.md]: @allella Link to the issue seems right.

**Comment by vkWeb** [comment]
@allella Please resolve the conflicts. Then remove JavaScript/TypeScript and replace it with just TypeScript there. 

Thanks for your work on this my friend ;)

**Comment by nik-john** [comment]
@all-contributors please add @allella  for doc

**Comment by nik-john** [comment]
@all-contributors please add @allella for doc
